src/CodeStyle/Core/Analyzers/Formatting/Formatter.cs

```diff
@@ -18,6 +18,11 @@ namespace Microsoft.CodeAnalysis.Formatting
     /// </summary>
     internal static class Formatter
     {
+        /// <summary>
+        /// The annotation used to mark portions of a syntax tree to be formatted.
+        /// </summary>
+        public static SyntaxAnnotation Annotation { get; } = new SyntaxAnnotation();
```

Comment:
note: i'm taking this approach because we have code that i'm sharing that does thing like add these annotations.  I could attempt to eitehr not share that code (which would mean ore shuffling around) or #ifdef it.  I chose to do this for expediency as i don't really see any issue with annotations being added.